.travis.yml

```diff
@@ -1,10 +1,37 @@
-language: php
+language:
+- php
+- node_js
 php:
-  - 7.1
+- 7.1
+node_js: "-'6.11'"
+install:
+- npm install
+- composer install
 script:
-  - composer update
-  - vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+- composer update
+- vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+cache:
+  directories:
+  - node_modules
+  - vendor
 after_success:
+  - ENVIRONMENT_NAME=$TRAVIS_BRANCH
+  - if [ "$TRAVIS_BRANCH" == "master" ]; then ENVIRONMENT_NAME=production; fi
   - vendor/bin/coveralls -v
   - vendor/bin/phpcs -n --standard=PSR1,PSR2 src/
   - vendor/bin/phpcbf src/
+before_deploy:
+- echo "All unit tests passed; Preparing to deploy $TRAVIS_BRANCH"
+deploy:
+- provider: script
+  skip_cleanup: true
+  script: "./scripts/travis-deploy.sh $TRAVIS_BRANCH"
```

Comment:
I think you want `"./scripts/travis-deploy.sh $ENVIRONMENT_NAME"`

Reply:
Fixed